packages/paste-website/src/components/Typography.tsx

```diff
@@ -31,8 +31,11 @@ export const InlineCode: React.FC = (props) => {
     <Box
       as="code"
       backgroundColor="colorBackground"
+      borderWidth="borderWidth10"
```

Comment:
Just checked this out on the quick start page and it LGTM.

One question: Now that the inline text is in a badge-like container, do we still need a [colon here](https://deploy-preview-2070--paste-docs.netlify.app/getting-started/quickstart#scripts)?

<img width="263" alt="Screen Shot 2021-11-23 at 9 35 29 AM" src="https://user-images.githubusercontent.com/18120906/143075501-5968195d-6e1c-4242-b8a7-0a36b2fc37e3.png">

Reply:
Hmm, I don't think the colon is necessary, but what would replace it? Might be confusing not having anything there to differentiate the script from the definition, since the inline code tags are used both before and after the colon.